supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -149,128 +143,85 @@ Use "Memory Overcommitment Manager (MOM)" for the first instance in a section, a
+*Description*: Red Hat Virtualization Host is the host in Red Hat Virtualization. It is a minimal operating system based on Red Hat Enterprise Linux, is distributed as an ISO file from the Customer Portal, and contains only the packages required for the machine to act as a host.
 
-Use "Red Hat Virtualization Host (RHVH)" for the first instance in a section. "RHVH" can be used in subsequent instances. Do not use "the Host" with a capital H. Do not use in Red Hat Enterprise Virtualization 3.x.
+Use "Red Hat Virtualization Host (RHVH)" for the first instance in a section. "RHVH" can be used in subsequent instances. Do not use "the Host" with a capital H.
 
 *Use it*: yes
 
 *Incorrect forms*: RHV-H, Red Hat Virtualization Hypervisor, RHV Host, the Host
 
-*See also*: xref:host-rhv[host], xref:red-hat-enterprise-virtualization-hypervisor[Red Hat Enterprise Virtualization Hypervisor]
+*See also*: xref:host-rhv[host]
 
 [discrete]
 [[red-hat-virtualization-manager]]
 ==== image:images/yes.png[yes] Red Hat Virtualization Manager (noun)
 *Description*: The Red Hat Virtualization Manager is a server that manages and provides access to the resources in the Red Hat Virtualization environment.
 
-Use "Red Hat Virtualization Manager" for version 4.x. Spell out in full for the first instance in a section. Use "the Manager" for subsequent instances. Do not use "the engine", which is the oVirt (upstream) term.
+Use "Red Hat Virtualization Manager". Spell out in full for the first instance in a section. Use "the Manager" for subsequent instances. Do not use "the engine", which is the oVirt (upstream) term.
 
 *Use it*: yes
 
 *Incorrect forms*: RHVM, RHV-M, RHV Manager, engine
 
-*See also*: xref:red-hat-enterprise-virtualization-manager[Red Hat Enterprise Virtualization Manager]
+*See also*:
 
 [discrete]
 [[resource-tab]]
 ==== image:images/yes.png[yes] resource tab (noun)
-*Description*: Hosts, virtual machines, storage, and other resources in Red Hat Virtualization can be managed by using their associated tab. For a visual example, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.1/html-single/introduction_to_the_administration_portal/#Graphical_User_Interface_elements[Graphical User Interface Elements] in the _Introduction to the Administration Portal_. The screen shot applies to Red Hat Virtualization 4.1 and earlier versions (including 3.x).
+*Description*: Hosts, virtual machines, storage, and other resources in Red Hat Virtualization can be managed by using their associated tab.
 
-You can refer to these tabs as just, for example, "the *Storage* tab", unlike the tabs in the details pane, which are always specified as such.
+You can refer to these tabs as just, for example, "the *Storage* tab".
```

Comment:
Maybe this one could be prescriptive instead of the 'you can', which seems a bit optional:

`Use the name of the tab when you refer to it, for example, "the *Storage* tab".`

something like this would remove the 'just' too, which we should avoid unless it's necessary.